Dockerfile

```diff
@@ -87,6 +87,7 @@ mv $GOPATH/bin/sqlflowserver /usr/local/bin && \
 mv $GOPATH/bin/repl /usr/local/bin && \
 (cd python/couler && python setup.py -q install) && \
 cd java/parser && \
+protoc --java_out=src/main/java --grpc-java_out=src/main/java/ --proto_path=src/main/proto/ src/main/proto/Parser.proto && \
```

Comment:
Is this line duplicated with the newly added line to test/java.sh?

Reply:
If so, it seems that we only need the line here, but not test/java.sh?

Reply:
> Is this line duplicated with the newly added line to test/java.sh?

They are not duplicated. The line here is used for building the release .jar file. The line in test/java.sh is used for testing.

This logic is consistent with having two `go generate ./...`s. One in Dockerfile, one in `scripts/test/units.sh`.